compute/resource_live_table.go

```diff
@@ -0,0 +1,221 @@
+	EnableEncryption bool   `json:"enable_encryption,omitempty" tf:"optional,default:false"`
+	EncryptionType   string `json:"encryption_type,omitempty" tf:"optional"`
+	KmsKey           string `json:"kms_key,omitempty" tf:"optional"`
+	CannedAcl        string `jsono:"canned_acl,omitempty" tf:"optional"`
+}
+
+type PipelinesClusterLogConf struct {
+	Dbfs *PipelinesDbfsStorageInfo `json:"dbfs,omitempty" tf:"optional"`
+	S3   *PipelinesS3StorageInfo   `json:"s3,omitempty" tf:"optional"`
+}
+
+type PipelinesInitScriptInfo struct {
+	Dbfs *PipelinesDbfsStorageInfo `json:"dbfs,omitempty" tf:"optional"`
+	S3   *PipelinesS3StorageInfo   `json:"s3,omitempty" tf:"optional"`
+	File *PipelinesLocalFileInfo   `json:"file,omitempty" tf:"optional"`
+}
+
+type PipelinesClusterAttributes struct {
+	SparkConf        map[string]string        `json:"spark_conf,omitempty"`
+	AwsAttributes    *PipelinesAwsAttributes  `json:"aws_attributes,omitempty"`
+	NodeTypeId       string                   `json:"node_type_id,omitempty"`
+	DriverNodeTypeId string                   `json:"driver_node_type_id,omitempty"`
+	SshPublicKeys    string                   `json:"ssh_public_keys,omitempty"`
+	CustomTags       map[string]string        `json:"custom_tags,omitempty"`
+	ClusterLogConf   *PipelinesClusterLogConf `json:"cluster_log_conf,omitempty"`
+	SparkEnvVars     map[string]string        `json:"spark_env_vars,omitempty"`
+	InstancePoolId   string                   `json:"instance_pool_id,omitempty"`
+}
+
+type PipelinesAutoScale struct {
+	MinWorkers int `json:"min_workers"`
+	MaxWorkers int `json:"max_workers"`
+}
+
+type PipelinesClusterSize struct {
+	NumWorkers int                 `json:"num_workers,omitempty"`
+	Autoscale  *PipelinesAutoScale `json:"autoscale,omitempty"`
+}
+
+type PipelinesNewCluster struct {
+	PipelinesClusterAttributes
+	PipelinesClusterSize
+}
+
+type PipelineCluster struct {
+	Label string `json:"label,omitempty"`
+	PipelinesNewCluster
+}
+
+type PipelinesMavenLibrary struct {
+	Coordinates string   `json:"coordinates"`
+	Repo        string   `json:"repo,omitempty"`
+	Exclusions  []string `json:"exclusions,omitempty"`
+}
+
+type NotebookLibrary struct {
+	Path string `json:"path"`
+}
+
+type PipelineLibrary struct {
+	Jar      string                 `json:"jar,omitempty"`
+	Maven    *PipelinesMavenLibrary `json:"maven,omitempty"`
+	Whl      string                 `json:"whl,omitempty"`
+	Notebook *NotebookLibrary       `json:"notebook,omitempty"`
+}
+
+type Filters struct {
+	Include []string `json:"include,omitempty"`
+	Exclude []string `json:"exclude,omitempty"`
+}
+
+type PipelineSpec struct {
+	Id            string            `json:"id,omitempty"`
+	Name          string            `json:"name,omitempty"`
+	Storage       string            `json:"storage,omitempty"`
+	Configuration map[string]string `json:"configuration,omitempty"`
+	Clusters      []PipelineCluster `json:"clusters,omitempty"`
```

Comment:
i'd advise to use `Cluster` type here and delete all fields that are not supported.